### PR TITLE
Dev workflow

### PR DESCRIPTION
**Pull request by Tharon-C**
This work by @jchansen and I improves the dev flow experience by cleaning up the directory structure, minimizing installation process, refreshing guide when lib changes and properly configured source maps.

* Update to Webpack2
* Build into a build folder and copy package.json for distrobution
* Remove MakeTheme.js and buildDocs.js from lib in favor of guide handling responsibility
* Removing need to include animate.css by rendering from lib

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "printWidth": 70,
```
Comment (lenards) [inline .prettierrc:2]: I would consider 80 columns, but that's not a huge deal. 

Python's [PEP 8 used 80](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

Comment (lenards) [reply .prettierrc:2]: JSX might make people want to push to 100. 

Totally open to discussing this.

**Review by lenards — APPROVED** [review]
:+1:

**Comment by cdosborn** [comment]
woot woot for prettier

**Comment by lenards** [comment]
💅

File: .prettierrc (hunk 1 above)
Comment (lenards) [reply .prettierrc:2]: Answer below ... ⬇️

Comment (Tharon-C) [reply .prettierrc:2]: I'll defer to your answer LOL
>source files for cyverse-ui will be included in the Style Guide (./guide within this repository). If you consider the guidelines for PEP8, comments have recommended column width of 72 for what I believe is a reason in that you'll be displaying it with a navigation bar to the one side. For PEP8, I think the reasoning is that you'll be commenting for a Docstring (__doc__) for a class, that puts you two levels indented (8 spaces, so 80-8 = 72).

Honestly, deep nesting is something I avoid and also will break up components into small pieces so 70 has been working well. I want to force inlining on long expressions as much as I can.

Comment (Tharon-C) [reply .prettierrc:2]: 72 works too

**Comment by lenards** [comment]
What is the `prettier` lifecycle? 

It is not included in `package.json` _scripts_ and it doesn't seem to be part of any automation for doing "format-check"s.

**Comment by lenards** [comment]
Added #92

**Pull request merged**